### ISSUE TITLE
Still maintained

### ISSUE THREAD

**Issue by rblalock** [zserge/anvil#129]
Just curious if this project is still maintained

**Comment by zserge** [comment]
@rblalock In theory - yes, I try to look at the issue tracker and PRs, but realistically - probably not  :pensive:  I don't do much android development anymore these days. Still, I would be happy to add maintainers if anyone wants to volunteer!

**Comment by rblalock** [comment]
haha.  Ok sweet thanks.  Ill do my best to contribute when I need some improvements.  I love this lib.  wouldn't want to see it die.

**Comment by sgrekov** [comment]
Yeah, it would be a shame for Android community if such a great lib will die(

**Comment by pardom** [comment]
Proposal to transfer to organization: https://github.com/zserge/anvil/issues/130

**Comment by sgrekov** [comment]
If anyone interested in Anvil, we made a fork https://github.com/inkremental/inkremental
It is rewritten in Kotlin, have such nice features as typesafe DSL and size qualifiers, supports AndroidX and latest support libraries, have Constraint Layout and Yoga Layout dsl  and other nice features!
